Extra/style2.css

```diff
@@ -0,0 +1,200 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;600;800&display=swap");
+
+:root {
+  --primary-color: #be185d;
+  --background-color: #fce7f2;
+  --secondary-color: #ffc72c;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
+}
+html,
+body {
+  height: 100vh;
+  width: 100vw;
+  background-color: var(--background-color);
+  /* background-attachment: fixed; */
+  margin: 0;
+  padding: 0;
+}
+
+/* Navbar */
+.navbar-container {
+  padding: 0.5rem 2rem;
+  width: 100%;
+  position: fixed;
+  z-index: 35;
+  font-weight: 600;
+  line-height: 30px;
+  box-shadow: 0px 10px 15px #ffcbe1;
+}
+
+.navbar-list-title {
+  margin-right: auto;
+}
+.navbar-title {
+  line-height: 20px;
+  font-size: 2rem;
+  font-weight: 600;
+}
+.navbar-list {
+  list-style: none;
+  display: flex;
+}
+
+.navbar-link {
+  text-decoration: none;
+  display: flex;
+}
+
+.navbar-list-items {
+  padding: 0rem 1rem;
+  font-size: 1.2rem;
+}
+.navbar-list-items a {
+  /* line-height: 10px; */
+  font-weight: 600;
+  color: var(--primary-color);
+}
+.beautiful {
+  font-weight: inherit;
+  color: var(--primary-color);
+}
+.sundar-logo {
+  position: relative;
+  bottom: 20px;
+}
+/* Container */
+.container {
+  /* overflow: hidden; */
+  /* margin: 5rem; */
+  position: relative;
+  top: 70px;
+}
+
+/* Umlaut */
+.dot {
+  color: black;
+}
+/* .secondary-dot {
+  color: var(--secondary-color);
+} */
+.stage {
+  /* border-bottom: 3px solid #444; */
+  display: flex;
+  height: 40px;
+  width: 100%;
+}
+.umlaut {
+  position: relative;
+  /* bottom: 22px;
+  right: 365px; */
+  align-self: flex-end;
+  animation-duration: 2s;
+  animation-iteration-count: infinite;
+  /* background-color: #fffe; */
+  height: 200px;
+  margin: 0 30rem 0rem auto;
+  transform-origin: bottom;
+  width: 200px;
+  animation-name: bounce-5;
+  animation-timing-function: ease;
+}
+
+@keyframes bounce-5 {
+  /* 0% {
+    transform: scale(1, 1) translateY(-100px);
+  } */
+  10% {
+    transform: scale(1.1, 0.9) translateY(0);
+  }
+  30% {
+    transform: scale(0.9, 1.1) translateY(-100px);
+  }
+  50% {
+    transform: scale(1, 1) translateY(0);
+  }
+  57% {
+    transform: scale(1, 1) translateY(-7px);
+  }
+  64% {
+    transform: scale(1, 1) translateY(0);
```

Comment:
Remove the extra folder if not getting used.

Reply:
removed.